packages/mobx-state-tree/package.json

```diff
@@ -48,31 +48,31 @@
         "dist/"
     ],
     "devDependencies": {
-        "@types/jest": "^24.0.11",
+        "@types/jest": "^26.0.3",
         "@types/node": "^12.0.2",
         "concat": "^1.0.3",
-        "coveralls": "^3.0.3",
+        "coveralls": "^3.1.0",
         "cpr": "^3.0.1",
-        "cross-env": "^6.0.3",
-        "jest": "^24.5.0",
-        "jest-junit": "^8.0.0",
+        "cross-env": "^7.0.2",
+        "jest": "^26.1.0",
+        "jest-junit": "^11.0.1",
         "lerna": "^3.13.1",
         "mobx": "^5.13.0",
-        "rollup": "^1.6.0",
+        "rollup": "^2.18.1",
         "rollup-plugin-commonjs": "^10.0.0",
-        "rollup-plugin-filesize": "^6.0.1",
+        "rollup-plugin-filesize": "^9.0.1",
         "rollup-plugin-node-resolve": "^5.0.0",
         "rollup-plugin-replace": "^2.1.0",
-        "rollup-plugin-terser": "^5.0.0",
+        "rollup-plugin-terser": "^6.1.0",
```

Comment:
dropped rollup < 2 and node < 10 support
https://github.com/TrySound/rollup-plugin-terser/commit/c87808afbdbfa299592b99bc57c47cdc36c73258